experimental/wgit/cli.py

```diff
@@ -5,18 +5,53 @@
 
 import argparse
 
+import experimental.wgit.weigit_api as weigit_api
+
 
 def main(argv=None):
-    desc = "WeiGit checkpoint tracking"
+    desc = "WeiGit: A git-like tool for Model checkpoint tracking"
     parser = argparse.ArgumentParser(description=desc)
 
     # flags
     parser.add_argument("-i", "--init", action="store_true", help="Initialize a weigit repository!")
+    parser.add_argument(
+        "-a",
+        "--add",
+        default="",
+        type=str,
+        metavar="FILE_PATH",
+        help="path to the checkpoint to be added (default: none)",
+    )
+    parser.add_argument("-s", "--status", action="store_true", help="Show the .wgit status")
+    parser.add_argument("-l", "--log", action="store_true", help="Show the log of .wgit")
```

Comment:
```suggestion
    parser.add_argument("-l", "--log", action="store_true", help="Show the repo's history log")
```

Reply:
log command should also take an optional file path to show history of a single file/dir.